ui/src/app/pages/demographic-survey.tsx

```diff
@@ -128,6 +154,15 @@ export const DemographicSurvey = (props: WithSpinnerOverlayProps) => {
           Submit
         </Button>
       </TooltipTrigger>
+      {redirectedFromSignIn && (
+        <Button
+          type='secondary'
+          onClick={handleMaybeLater}
```

Comment:
The function name `handleMaybeLater` says what the function *is* but I'd prefer a more descriptive name about what it *does*, which would allow us to read this line and know a little something about the behavior.

For a name idea... `dismissAndContinue` ?